cpr/ssl_options.cpp

```diff
@@ -0,0 +1,12 @@
+#include "cpr/ssl_options.h"
+
+namespace cpr {
+
+VerifySsl::VerifySsl(bool verify)
+          : verify_{verify} {}
```

Comment:
Can you put this on the above line? The format checker is [complaining](https://travis-ci.org/whoshuu/cpr/jobs/192078769#L7879-L7880). I'll write up instructions for how to do this manually at some point, and to also fail the build if the format checker doesn't pass.